src/pkg/layout/split.go

```diff
@@ -18,30 +18,37 @@ import (
 )
 
 // splitFile will split the file into chunks and remove the original file.
-func splitFile(srcPath string, chunkSize int) error {
+func splitFile(srcPath string, chunkSize int) (err error) {
 	srcFile, err := os.Open(srcPath)
 	if err != nil {
 		return err
 	}
-	defer srcFile.Close()
```

Comment:
I know the existing code closes it later on, but I believe we should keep this one and delete the other

Reply:
Fixed in latest